drivers/hmis/app/models/hmis/hud/client.rb

```diff
@@ -117,6 +117,15 @@ class << self
     end
   end
 
+  scope :older_than, ->(age, or_equal: false) do
+    target_dob = Date.today - (age + 1).years
+    target_dob = Date.today - age.years if or_equal == true
+
+    where(c_t[:dob].lt(target_dob))
+  end
+
+  scope :adults, -> { older_than(18, or_equal: true) }
```

Comment:
I'd prefer to avoid adding more age math into the code base, since it already has plenty. What about adding the scope to this [shared concern](https://github.com/greenriver/hmis-warehouse/blob/292076714bdb015c56bcf6af8d057314a5f7ce7c/app/models/concerns/hud_concerns/client.rb#L67) and making use of the math we already have?

```
scope :adults, -> age_group(start_age: 18)
```